runtime/src/bank.rs

```diff
@@ -1905,6 +1921,28 @@ impl Bank {
         });
     }
 
+    fn fill_sysvar_cache(&mut self) {
+        let mut sysvar_cache = self.sysvar_cache.write().unwrap();
+        #[allow(deprecated)]
+        for id in &[
+            sysvar::clock::id(),
+            sysvar::epoch_schedule::id(),
+            sysvar::fees::id(),
+            sysvar::recent_blockhashes::id(),
+            sysvar::rent::id(),
+            sysvar::rewards::id(),
+            sysvar::slot_hashes::id(),
+            sysvar::slot_history::id(),
+            sysvar::stake_history::id(),
```

Comment:
Would be nice if this is coalesced with our other set of sysvar accounts in the `is_sysvar` function.

Reply:
I tried having a `get_all_sysvar_ids()` method and then searching through that. But the consequence is that the list of all ids needs to be created (which means copied) for every `is_sysvar` check, because `sysvar::id()` is not const and thus can't be used for static variable initialization.

Reply:
Yea, it would have to be a list created by lazy_static!() maybe. There is some perf overhead to that too though.

Reply:
Done

Reply:
I think we use it in `is_writable()` which we want to call before actually loading the accounts from accountsdb.